python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -240,6 +240,15 @@ def deblend(mExposure, footprint, config):
     ymin = bbox.getMinY()
     centers = [np.array([peak.getIy()-ymin, peak.getIx()-xmin], dtype=int) for peak in footprint.peaks]
 
+    # Choose whether or not to use the improved spectral initialization
+    if config.setSpectra:
+        if config.maxSpectrumCutoff == -1:
+            spectrumInit = True
+        else:
+            spectrumInit = len(centers) * bbox.getArea() < config.maxSpectrumCutoff
+    else:
+        spectrumInit = config.setSpectra
```

Comment:
I think in this case this is always `False`, right?  I think it should just be set to `False`.

Reply:
Yes. Originally I tried to use `True` to always use `setSpectra`, `False` to never use it, and `None` to check `maxSpectrumCutoff`, but the stack won't let me use `None` as the default for a `bool` (which makes sense, but :( ).